src/Tags/Rotate.php

```diff
@@ -3,12 +3,18 @@
 namespace Statamic\Tags;
 
 use Statamic\Support\Arr;
+use Statamic\View\State\ResetsState;
 
-class Rotate extends Tags
+class Rotate extends Tags implements ResetsState
 {
     protected static $aliases = ['switch'];
     protected static $counts = [];
 
+    public function resetState()
```

Comment:
```suggestion
    public static function resetStaticState()
```

See other comment for explanation.